pkg/agent/plugin/svidstore/repository.go

```diff
@@ -0,0 +1,21 @@
+package svidstore
+
+type Repository struct {
+	SVIDStores []SVIDStore
```

Comment:
I think, like NodeAttestors, the SVIDStore plugins are used by name (for example, the svidstore service immediate takes these and puts them into a map). For that reason, I think we should model this Repository like the NodeAttestor or NodeResolver repositories:
```suggestion
	SVIDStores map[string]SVIDStore
```
See https://github.com/spiffe/spire/blob/327a0dbcbf1ece82bf3dac83b4144b69faddfded/pkg/server/plugin/nodeattestor/repository.go#L4